enterprise/models.py

```diff
@@ -366,7 +366,7 @@ def enterprise_customer_identity_provider(self):  # pylint: disable=invalid-name
     @property
     def identity_provider(self):
         """
-        Return the identity providers associated with this enterprise customer.
+        Return the identity provider id associated with this enterprise customer.
```

Comment:
shouldn't it be `Return the identity **providers** id associated with this enterprise customer.`

Reply:
its returning id of only *one* identity_provider(i.e first one) so think it should 
`Return the identity provider id associated with this enterprise customer.`
or
`Return the identity provider's id associated with this enterprise customer.`

Reply:
Is this where the check for a "default" IdP would happen?

Reply:
Also this seems like a good place to add some logging information -- for example what if no IdP is found.

Reply:
Yes, it seems a good place for a default check. added logger.